app/assets/javascripts/admin/models/tl3-requirements.js

```diff
@@ -12,6 +12,11 @@ export default EmberObject.extend({
     return Math.round((minDaysVisited * 100) / timePeriod);
   },
 
+  @discourseComputed("num_topics_replied_to", "min_topics_replied_to")
+  capped_topics_replied_to(numReplied, minReplied) {
+    return numReplied === 2 * minReplied;
```

Comment:
I think `>=` would be more correct here.

```suggestion
    return numReplied >= minReplied;
```

Reply:
If I'm doing a +1, then `>` would be correct, no?

Reply:
I'm always off by one :P